scripts/globals/abilities/hamanoha.lua

```diff
@@ -0,0 +1,18 @@
+-----------------------------------
+-- Ability: Hamanoha
+-- Description	Lowers accuracy, evasion, magic accuracy, magic evasion and TP gain for demons.
```

Comment:
Sneaky tab

Reply:
took care of it and went thru others to make sure no more sneaky tabs showed up